docs/enterprise-v2.7.md

```diff
@@ -34,6 +34,33 @@ Key highlights of this release:
 3. For Kubernetes users, please follow [the guide](/k8s-helm#v5-to-v6) carefully since a few timeplusd built-in users are removed in the new helm chart, and you can configure ingress for Appserver and Timeplusd independently.
 
 ## Releases
+### 2.7.6 {#2_7_6}
+Released on 04-30-2025. Installation options:
+* For Linux or Mac users: `curl https://install.timeplus.com/2.7 | sh` [Downloads](/release-downloads#2_7_6)
```

Comment:
Need to need to run `autogenerate-downloads.sh` to update the `docs/release-downloads.md` based on S3 files. Otherwise this link won't work